scripts/time_to_peak.R

```diff
@@ -0,0 +1,37 @@
+#time to peak delF response for 1 pulse
+#open PlotGCaMP_multi csv file
```

Comment:
I'd just indicate here that this is intended for multi-pulse files. You could potentially make it more flexible and calculate for 2nd, 3rd etc.. but that's optional.

Reply:
done